packages/integration-tests/test/standardExitTest.js

```diff
@@ -200,9 +217,18 @@ describe('Standard Exit tests', async () => {
       // Get the exit data
       const utxoToExit = bobUtxos[0]
       const exitData = await childChain.getExitData(utxoToExit)
-      assert.hasAllKeys(exitData, ['txbytes', 'sigs', 'proof', 'utxo_pos'])
+      assert.hasAllKeys(exitData, ['txbytes', 'proof', 'utxo_pos'])
 
-      let receipt = await rootChain.startStandardExit(
+      try {
+        const addTokenCall = await rootChain.addToken(transaction.ETH_CURRENCY, { from: bobAccount.address, privateKey: bobAccount.privateKey })
```

Comment:
Can we use the same `rootChain.hasToken()` check here as above?
Also, I think that this should go into the `before()` block, as it's really part of the setup rather than the test itself

Reply:
to your second point, i agree but then I also like seeing the addToken call right before the startStandardExit. For anyone that reads this test in the future maybe its more explicit of the chronological steps that need to happen when calling the exit.